### ISSUE TITLE
problem

### ISSUE THREAD

**Issue by khaibnd** [spyder-ide/spyder#5400]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "/usr/local/lib/python2.7/dist-packages/spyder/plugins/ipythonconsole.py", line 890, in run_script
    self.execute_code(line, current_client, clear_variables)
  File "/usr/local/lib/python2.7/dist-packages/spyder/plugins/ipythonconsole.py", line 936, in execute_code
    sw.execute(to_text_string(to_text_string(lines)))
  File "/usr/local/lib/python2.7/dist-packages/qtconsole/history_console_widget.py", line 47, in execute
    source, hidden, interactive)
  File "/usr/local/lib/python2.7/dist-packages/qtconsole/console_widget.py", line 632, in execute
    self._execute(source, hidden)
  File "/usr/local/lib/python2.7/dist-packages/qtconsole/frontend_widget.py", line 294, in _execute
    msg_id = self.kernel_client.execute(source, hidden)
AttributeError: 'NoneType' object has no attribute 'execute'

## Version and main components

* Spyder Version: 3.2.3 
* Python Version: 2.7.12
* Qt Versions: 4.8.7, PyQt4 (API v2) 4.11.4 on Linux

## Dependencies
```
pyflakes >=0.5.0  :  1.5.0 (OK)
pycodestyle >=2.3 :  2.3.1 (OK)
pygments >=2.0    :  2.2.0 (OK)
pandas >=0.13.1   :  0.17.1 (OK)
numpy >=1.7       :  1.13.1 (OK)
sphinx >=0.6.6    :  1.6.4 (OK)
rope >=0.9.4      :  0.10.7 (OK)
jedi >=0.9.0      :  0.11.0 (OK)
psutil >=0.3      :  5.3.1 (OK)
nbconvert >=4.0   :  5.3.1 (OK)
sympy >=0.7.3     :  None (NOK)
cython >=0.21     :  None (NOK)
qtconsole >=4.2.0 :  4.3.1 (OK)
IPython >=4.0;<6.0:  5.5.0 (OK)
pylint >=0.25     :  1.5.2 (OK)

```

**Comment by ccordoba12** [comment]
This is a duplicate of issue #5053 and it'll be fixed in Spyder 3.2.4

**Comment by khaibnd** [comment]
Thanks
